### ISSUE TITLE
data file not found in such condition

### ISSUE THREAD

**Issue by richard0326** [andy-yun/pytorch-0.4-yolov3#72]
if I increase dataset over 100...

I get No such file or directory error.
but the file exist...
I think it is some kind of threading problem.. not sure

maybe file is no closed. and trying to reopen.
I will upload the error screen. 
![1](https://user-images.githubusercontent.com/18134703/58861732-fce6d900-86e9-11e9-9044-3c89fb94f970.PNG)

if I reduce data it does not fail to train or valid on this error

**Comment by andy-yun** [comment]
@richard0326 You should change batch and subdivision size to 1 as follows when testing:

```
[net]
# Testing
batch=1
subdivisions=1
# Training
# batch=8
# subdivisions=4
```

**Comment by richard0326** [comment]
You mean ?
```
[net]
# Testing
batch=1
subdivisions=1[net]
# Training <- not Testing
batch=1
subdivisions=1
```

I have Tried this but I get 0 value error so I increased batch to 8, and subdivisions to 4
But if I do this I get upper error occured before...

배치를 1 서브디비전을 1로 넣어서 할 경우, 0 값이 나오는 에러가 발생합니다. 여기에 있는 깃을 보고 배치를 늘리면 해결할 수 있음을 알게되어 배치를 8로 서브디비전을 4로 늘려서 해결했었는데, train 중간에 있는 evaluation에서 에러가 발생하네요. -n을 하고 detection을 따로 돌려도 위와 같은 에러가 발생합니다.

**Comment by andy-yun** [comment]
@richard0326 
comment was messed. T-T
When training mode, you should comment the two lines : batch=1 and subdivisions=1, uncomment two lines: batch=8 and subdivisions=4.
If you want to avoid the data error, you could substitute batchsize to 1 in 54 line of train.py.
I will investigate the source code when batchsize > 1 on test mode.
Or you should check the test file list whether files are duplicated. If files are duplicated, the file open error may occurred.

**Comment by richard0326** [comment]
Umm.. I solved the problem of fileOpen.... it was my mistake I some how erase the file... sorry...
파일 경로 설정에 문제가 있었나 봅니다....

Some how I get Zero value error like this, when I uncomment two lines: batch=8 and subdivisions=4.
주석 달지 않고하니까 되네요... 어떻게 된걸까요?

**Comment by richard0326** [comment]
anyway the problem was my file directory so I'm closing the issue~
Thank you for the help
